docs/components/NavigationBottom.md

```diff
@@ -19,130 +18,66 @@ description: Bottom navigation bars allow movement between primary destinations
-
-In `layout.xml`:
-
-```xml
-<LinearLayout
-    xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:app="http://schemas.android.com/apk/res-auto"
-    android:layout_width="match_parent"
-    android:layout_height="wrap_content">
-
-  <com.google.android.material.bottomnavigation.BottomNavigationView
-      android:id="@+id/bottom_navigation"
-      android:layout_width="match_parent"
-      android:layout_height="wrap_content"
-      app:menu="@menu/bottom_navigation_menu" />
-
-</LinearLayout>
+    private data class NavigationItem(
+    @StringRes val titleResId: Int,
+    @DrawableRes val iconResId: Int
+)
+
+val items = listOf(
+    R.string.component_bottom_navigation_coffee to R.drawable.ic_coffee,
+    R.string.component_bottom_navigation_cooking_pot to R.drawable.ic_cooking_pot,
+    R.string.component_bottom_navigation_ice_cream to R.drawable.ic_ice_cream,
+    R.string.component_bottom_navigation_restaurant to R.drawable.ic_restaurant,
+    R.string.component_bottom_navigation_favorites to R.drawable.ic_heart
+)
+
+var selectedItemIndex by remember { mutableStateOf(0) }
+
+OdsBottomNavigation(
+    items = items.mapIndexed { index, item ->
+        OdsBottomNavigationItem(
+            icon = OdsBottomNavigationItemIcon(
+                painter = painterResource(id = item.first),
+                contentDescription = ""
+            ), // contentDescription is empty cause TalkBack already read the item's label
+            label = stringResource(id = item.second),
+            selected = selectedItemIndex == index,
+            onClick = {
+                selectedItemIndex = index
+                doSomething()
+            }
+        )
+    }
+)
 ```
 
-In `bottom_navigation_menu.xml` inside a `menu` resource directory:
-
-```xml
-<menu xmlns:android="http://schemas.android.com/apk/res/android">
-  <item
-      android:id="@+id/page_1"
-      android:enabled="true"
-      android:icon="@drawable/ic_favorite"
-      android:title="@string/favorites"/>
-  <item
-      android:id="@+id/page_2"
-      android:enabled="true"
-      android:icon="@drawable/ic_music"
-      android:title="@string/music"/>
-  <item
-      android:id="@+id/page_3"
-      android:enabled="true"
-      android:icon="@drawable/ic_places"
-      android:title="@string/places"/>
-  <item
-      android:id="@+id/page_4"
-      android:enabled="true"
-      android:icon="@drawable/ic_news"
-      android:title="@string/news"/>
-</menu>
-```
-
-In code:
+#### OdsBottomNavigation API
 
-```kotlin
-bottomNavigation.selectedItemId = R.id.page_2
-```
+Parameter | Default&nbsp;value | Description
+-- | -- | --
+`modifier: Modifier` | `Modifier` | `Modifier` applied to the bottom navigation
+`items: List<OdsBottomNavigationItem>` | | Items displayed into the bottom navigation
```

Comment:
`modifier` should be below `items`, probably fixed in the other PR.

Reply:
Yes, this is the case in next PR.